pkg/kubectl/cmd/create.go

```diff
@@ -234,7 +234,11 @@ func RunCreateSubcommand(f *cmdutil.Factory, cmd *cobra.Command, out io.Writer,
 	}
 
 	if useShortOutput := options.OutputFormat == "name"; useShortOutput || len(options.OutputFormat) == 0 {
-		cmdutil.PrintSuccess(mapper, useShortOutput, out, mapping.Resource, options.Name, "created")
+		successMsg := "created"
+		if options.DryRun {
+			successMsg = fmt.Sprintf("%s %s", successMsg, "(DRY RUN)")
```

Comment:
> would "created (dry run)" be less shouty?
I think so, I can also go ahead and change it anywhere else that is printing it in caps.

> are there any other places we're calling PrintSuccess from commands that take a --dry-run arg we need to qualify the output?

hm, there are a few more commands, other than `oc create` subcommands, however some of these, such as `kube rolling-update` [print a different output](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/rollingupdate.go#L329-L345) instead of showing the normal success message when using a `--dry-run` flag. Do you think we should still include the words `(dry run)` in there somewhere?